src/site/content/en/web-dev-basics-one/additional-html-elements/index.md

```diff
@@ -0,0 +1,164 @@
+
+## Document metadata
+
+```html
+<title>Sarah's Favorite Food Recipes</title>
+```
+
+Anytime you type in a website URL, there's a [`<title>`](https://developer.mozilla.org/docs/Web/HTML/Element/title) name that can be read in the browser bar or web page tab. It's the title name you see given for a web page. This element is important and is used by a search engine to display a list of related web pages in search results. The title length can vary from short and descriptive, to longer and more descriptive. 
+
+Scenario: you have a web page you're thinking about, but can't remember the specific website URL. Type in the keywords in a search engine. The search engine helps track down the web page that you're looking for, and you can view the `<title>` name appearing in search.
+
+## Embedded content elements
+
+In addition to text content, there are a variety of additional content elements to use. 
+
+The `<iframe>` element
+
+```html
+<iframe src="https://www.wikipedia.org/" title="Wikipedia"></iframe>
+```
+
+When finishing shopping for items online and clicking on your payment option, such as Paypal or Apple Pay, these features are commonly added to a web page with an [`<iframe>`](https://developer.mozilla.org/docs/Web/HTML/Element/iframe). Seeing a map online to search for a local business is another common experience. These types of user experiences on a web page can be added with an iframe.  In the example above, you see the Wikipedia URL within an iframe, titled as "Wikipedia." 
+
+The `<iframe>` element allows you to insert content from another source and embed a frame within a web page. It creates a rectangular shaped frame and displays content in the browser. A frame allows for the presentation of a window shaped layout within an `<iframe>` element. It's a powerful way to add content to your web page to enhance the experience. 
+
+{% Aside %}
+Think about which types of web page content you would want to embed onto another page, using the `<iframe>` and `<source>`. What would it look like?
+{% endAside %}
+
+## Form elements
+
+```html
+<progress max="100" value="30"> 30% </progress>
+```
+
+When watching a longer video, lecture, or filling out an extensive application, having a visual progress bar can be helpful to track your progress. The [`<progress>`](https://developer.mozilla.org/docs/Web/HTML/Element/progress) element is helpful for these types of scenarios. 
+
+This element is depicted as a visual bar with a background color. The visual bar can range in size and background color. With the progress bar, you can optionally use the `max` and `value` attributes. The `max` attribute sets the floating point number and the `value` attribute notes how much progress has been made on a task.  
+
+## Scripting
```

Comment:
Not sure if `Scripting` is the best title here. Shouldn't it be `Drawing` or something along those lines since we're talking about the canvas API?